Dockerfile

```diff
@@ -45,6 +48,6 @@ RUN chown -R errbit:errbit /app
 
 USER errbit
 
-HEALTHCHECK CMD curl --fail http://localhost:$PORT/users/sign_in || exit 1
+HEALTHCHECK CMD curl --fail "http://$(/sbin/ip route | /usr/bin/awk '/src/{print $NF}'):8080/users/sign_in" ||  exit 1
```

Comment:
I think this is good. I can't think of any reason to need to change the bound port when running in a container.